### PR TITLE
[12.0][MIG] purchase_requisition_auto_rfq

### PR DESCRIPTION
**Pull request by Jerther**
Migrated module! :sweat_smile:

### CODE REVIEW

**Comment by Jerther** [comment]
Only a couple of line lengths "problems" remain in the Travis build report. They're quite hard to shorten without hurting the code readability, so I leave it to the reviewers to decide... :smile:

File: purchase_requisition_auto_rfq/tests/test_purchase_requisition.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        self._kitchenset = self.env.ref('purchase_requisition_auto_rfq.kitchenset')
+        self._blankets = self.env.ref('purchase_requisition_auto_rfq.blankets')
+        self._tarpaulin = self.env.ref('purchase_requisition_auto_rfq.tarpaulin')
+
+    def _init_requisitions(self):
+        uom = self._kitchenset.uom_po_id
+        self._requisition1 = self.env['purchase.requisition'].create({
+            'name': "PR01",
+            'line_ids': [
+                (0, False, {'product_id': self._kitchenset.id, 'product_qty': 10, 'product_uom_id': uom.id}),
```
Comment (ps-tubtim) [inline purchase_requisition_auto_rfq/tests/test_purchase_requisition.py]: You should change this code for fix flake8.
```suggestion
                (0, False, {'product_id': self._kitchenset.id,
                            'product_qty': 10, 'product_uom_id': uom.id}),
```

File: purchase_requisition_auto_rfq/tests/test_purchase_requisition.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                (0, False, {'product_id': self._blankets.id, 'product_qty': 100, 'product_uom_id': uom.id}),
```
Comment (ps-tubtim) [inline purchase_requisition_auto_rfq/tests/test_purchase_requisition.py]:
```suggestion
                (0, False, {'product_id': self._blankets.id,
                            'product_qty': 100, 'product_uom_id': uom.id}),
```

File: purchase_requisition_auto_rfq/tests/test_purchase_requisition.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            ]
+        })
+
+        self._requisition2 = self.env['purchase.requisition'].create({
+            'name': "PR02",
+            'line_ids': [
+                (0, False, {'product_id': self._tarpaulin.id, 'product_qty': 10, 'product_uom_id': uom.id}),
```
Comment (ps-tubtim) [inline purchase_requisition_auto_rfq/tests/test_purchase_requisition.py]:
```suggestion
                (0, False, {'product_id': self._tarpaulin.id,
                            'product_qty': 10, 'product_uom_id': uom.id}),
```

File: purchase_requisition_auto_rfq/tests/test_purchase_requisition.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                (0, False, {'product_id': self._blankets.id, 'product_qty': 100, 'product_uom_id': uom.id}),
```
Comment (ps-tubtim) [inline purchase_requisition_auto_rfq/tests/test_purchase_requisition.py]:
```suggestion
                (0, False, {'product_id': self._blankets.id,
                            'product_qty': 100, 'product_uom_id': uom.id}),
```

**Review by flotho — APPROVED** [review]
Hi,

Thanks for this awesome migration.
Functionnal test done

LGTM

**Review by gaelTorrecillas — APPROVED** [review]
I tested with odoo 12 (OCB) and it worked.

**Review by ps-tubtim — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
squash commits from 5b1be08 to 1d5a9bc

**Comment by Jerther** [comment]
@HviorForgeFlow done

**Comment by Jerther** [comment]
Phew! I can finally get on with my life :stuck_out_tongue_winking_eye:

:beers:

**Pull request merged**